### PR TITLE
Change project name to UniLogin in Circle CI status badge

### PR DESCRIPTION
**Pull request by JustynaBroniszewska**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![CircleCI](https://circleci.com/gh/UniversalLogin/UniversalLoginSDK/tree/master.svg?style=svg)](https://circleci.com/gh/UniversalLogin/UniversalLoginSDK/tree/master)
+[![CircleCI](https://circleci.com/gh/UniLogin/UniLogin/tree/master.svg?style=svg)](https://circleci.com/gh/UniLogin/UniLogin/tree/master)
```
Comment (JustynaBroniszewska) [inline README.md:1]:
```suggestion
[![UniLogin](https://circleci.com/gh/UniLogin/UniLogin/tree/master.svg?style=svg)](https://circleci.com/gh/UniLogin/UniLogin/tree/master)
```

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-[![CircleCI](https://circleci.com/gh/UniversalLogin/UniversalLoginSDK/tree/master.svg?style=svg)](https://circleci.com/gh/UniversalLogin/UniversalLoginSDK/tree/master)
+[![UniLogin](https://circleci.com/gh/UniLogin/UniLogin/tree/master.svg?style=svg)](https://circleci.com/gh/UniLogin/UniLogin/tree/master)
 
 ![Universal-Login](./docs/source/static/logo.png)
```
Comment (xitronix) [inline README.md:3]: What about here? Can we change this too?

Comment (JustynaBroniszewska) [inline README.md:3]:
```suggestion
![UniLogin](./docs/source/static/logo.png)
```

Comment (JustynaBroniszewska) [reply README.md]: Yes, we can 😉

**Pull request merged**